tests/operations.go

```diff
@@ -364,6 +361,17 @@ func (ops *Operations) IsPVCDeleted(pvcName string) bool {
 	return false
 }
 
+// IsPodDeletedEventually checks if the pod is deleted or not eventually
+func (ops *Operations) IsPodDeletedEventually(podName string) bool {
```

Comment:
IMO this should accept namespace as a argument

Reply:
Done